tests/integration/devfile/utils/utils.go

```diff
@@ -532,7 +532,14 @@ func validateContainerExecListDir(odoV1Watch OdoV1Watch, odoV2Watch OdoV2Watch,
 			}
 			cliRunner := runner.(helper.CliRunner)
 			podName := cliRunner.GetRunningPodNameByComponent(odoV2Watch.CmpName, project)
+
+			helper.WaitForCmdOut("oc", []string{"exec", podName, "--namespace", project,
```

Comment:
This will fail for `kubernetes`, no? We have `WaitForRunnerCmdOut` function for oc and kubernetes both.